book/src/07_web_api/openapi/core_backend_api.yaml

```diff
@@ -1100,7 +1100,7 @@ components:
             * True - **MUST** include a note.
         map:
           type: array
-          item: string
+          item: object
```

Comment:
If its being suggested it be `{"name":<name>, "desc":<desc>}` then this should be defined here.
`object` doesn't tell an API consumer what they should expect.

Also, the `example` is now wrong as well.